monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/frontends.dashboard.py

```diff
@@ -0,0 +1,50 @@
+from grafanalib.core import (
+    Dashboard, Graph, Row, Target
+)
+from grr_grafanalib_dashboards.util import add_data_source
+from grr_grafanalib_dashboards.config import PANELS
+
+GRR_COMPONENT = "frontend"
+
+dashboard = Dashboard(
+    title="{}s Dashboard".format(GRR_COMPONENT).title(),
+    rows=[
+        Row(panels=[
+            panel(GRR_COMPONENT) for panel in PANELS
+        ]
+        ),
+        Row(panels=[
+          Graph(
+            title="QPS",
+            targets=[
+                Target(
+                    expr='sum(rate({}_request_count_total[1m]))'.format(GRR_COMPONENT),
+                    legendFormat="Requests",
+                ),
+            ],
+            ),
+            Graph(
+            title="Request Latency Rate",
+            targets=[
+                Target(
+                    expr='sum(rate({0}_request_latency_sum[5m])) / sum(rate({0}_request_latency_count[5m]))'.format(GRR_COMPONENT),
```

Comment:
Please see my comment above. I think it makes sense to name the metric literally.